requirements-server.txt

```diff
@@ -8,7 +8,7 @@
 -r requirements.dev.txt
 
 # For worker manager
-boto3==1.9.228
+boto3~=1.14.33
```

Comment:
Are these part of this PR?

Reply:
Yes, it's upgrading a boto3 version. It's suggested by pycharm.